frame/babe/src/lib.rs

```diff
@@ -18,7 +18,7 @@
 //! from VRF outputs and manages epoch transitions.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![forbid(unused_must_use, unsafe_code, unused_variables, unused_must_use)]
+#![warn(unused_must_use, unsafe_code, unused_variables, unused_must_use)]
```

Comment:
is this intentional (or a result of debugging)?

Reply:
Intentional. Using `forbid` may cause issues with:
- Quick development cycle
- Build randomly breaking on rustc upgrades

We did try that in the past, but it was doing more harm than good.